src/components/executive-education-2u/ExecutiveEducation2UError.jsx

```diff
@@ -48,9 +76,21 @@ const ExecutiveEducation2UError = ({ failureReason, httpReferrer }) => {
                 global.location.href = httpReferrer;
               }}
             >
-              Return to your learning platform
+              Return to dashboard
             </Button>
           )}
+          {showHelpfulLink(failureReason) && (
+            <div>
+              <span className="executive-education-2u-error-link-description">
+                Helpful Link:
+              </span>
+              <span className="executive-education-2u-error-link-box">
+                <Button variant="link" size="sm">
```

Comment:
Would recommend just rendering `ContactAdminMailto` on its own (not within a `Button`, since the button is already an interactive element with a larger click target, and clicking the button itself doesn't do anything (only clicking hyperlink from `MailtoLink` works), which is a bit of a usability issue).

```jsx
<span className="executive-education-2u-error-link-box"> // assumes this class name gives the `font-size: $font-size-xs;` style property.
  <ContactAdminMailto />
</span>
```